DATASETS.md

```
@@ -25,10 +25,11 @@ Expected `col_name | col_type | col_description` for each per-game-compiled **se
 | [rosters](#rosters) | one row per rostered athlete per season (deduped) | ~63 | `espn_cfb_rosters` |
 | [betting](#betting) | one row per game | 9 | `espn_cfb_betting` |
 | [schedules](#schedules) | one row per game | 34 | `espn_cfb_schedules` |
+| [cfb_schedules](#cfb_schedules) ‡ | one row per game (all divisions) | 41 | `cfb_schedules` |
 | [linescores](#linescores) | one row per (team, period) | 5 | `espn_cfb_linescores` |
 | [power_index](#power-index) | one row per team (or per game) | 22 | `espn_cfb_power_index` |
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)